packages/drivers/tinylicious-driver/src/insecureTinyliciousUrlResolver.ts

```diff
@@ -93,7 +94,8 @@ export class InsecureTinyliciousUrlResolver implements IUrlResolver {
 
 /**
  * Creates a Routerlicious {@link @fluidframework/core-interfaces#IRequest}.
- * @internal
+ * @legacy
+ * @alpha
  */
 export const createTinyliciousCreateNewRequest = (documentId?: string): IRequest => ({
```

Comment:
Do we know of any reason this one shouldn't just be `@public` if it going to be supported?
The Insecure things seem like they should stay more locked down. I could see placing those under a test utils export, but that doesn't seem right for `createTinyliciousCreateNewRequest`.
Maybe @ChumpChief or @CraigMacomber have some thoughts. (It would be nice to not have this added /legacy export here.)

Reply:
We don't currently expose the request pattern or url resolvers on the public API, and my hope is that we can make all of that go away (rather than public).  If we were going to make the functionality public, I think we'd probably go with createAzureCreateNewRequest instead which fills the same role for AzureClient used in "local" mode.

Reply:
So, what should our opinion be for Loop's use: https://office.visualstudio.com/OC/_search?action=contents&text=getTinyliciousLoopWebServiceDataFetcher&type=code&filters=ProjectFilters%7BOC%7DRepositoryFilters%7Boffice-bohemia%7D&pageSize=25&result=DefaultCollection/OC/office-bohemia/GBmaster//packages/loop-web-service/src/container/LoopWebServiceDataFetcher.ts

Reply:
I think tagging legacy for now is prob ok and they keep using it, with the hope that we take on the work to get rid of the URL resolver and eliminate it altogether.  I think that's probably a smoother path to unification than making a new API public only to take it away again later.

Reply:
Then he is a consideration to limit the exposure and make it clear that some of these things are test only: https://github.com/microsoft/FluidFramework/pull/23284